src/diagnostics/define_diagnostics.jl

```diff
@@ -16,7 +16,7 @@ function define_diagnostics!(land_model)
         long_name = "Albedo",
         standard_name = "albedo",
         units = "",
-        comments = "The fraction of incoming radiation reflected by the land surface.",
+        comments = "The fraction of downwelling radiation reflected by the land surface.",
```

Comment:
Being strict here, we need to define which albedo we're talking about about, given that at the moment we have 3 albedos (SW, PAR, and NIR). This one should be "SW_albedo", or something like that. It is "The fraction of downwelling SHORTWAVE radiation reflected by the land surface."

Reply:
I fixed this! I also added SW albedo as a diagnostic for the land model (currently it was just an option for the bucket). Now we will save that instead of SW_u.